ert_shared/services/_webviz_ert_main.py

```diff
@@ -73,4 +96,8 @@ def run_webviz_ert():
 
 
 if __name__ == "__main__":
-    run_webviz_ert()
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--experimental_mode", type=str, default="False")
+    args = parser.parse_args()
+
+    run_webviz_ert(args.experimental_mode.lower() == "true")
```

Comment:
This is a bit strange to me, it is not equivalent to:

```python
    parser = argparse.ArgumentParser()
    parser.add_argument("--experimental_mode", action="store_true")
    args = parser.parse_args()

    run_webviz_ert(args.experimental_mode)
```
Also, I would also add:

```python
    parser.add_argument("--experimental_mode", "-e",  action="store_true")
```

Sorry if I am missing the obvious reason why we are required to convert the command line argument to a string and then back to a boolean.